arangod/Aql/AqlValue.cpp

```diff
@@ -1339,64 +1068,52 @@ int AqlValue::Compare(velocypack::Options const* options, AqlValue const& left,
         return 1;
       }
       return 0;
-    }
   }
   return 0;
 }
 
-AqlValue::AqlValue() noexcept {
-  // construct a slice of type None
-  // we will simply zero-initialize the two 64 bit words
-  _data.words[0] = 0;
-  _data.words[1] = 0;
-
-  // VPACK_INLINE must have a value of 0, and VPackSlice::None must be equal
-  // to a NUL byte too
-  static_assert(AqlValueType::VPACK_INLINE == 0,
-                "invalid value for VPACK_INLINE");
```

Comment:
Is this `static_assert` kept somewhere (probably in the header file)?
It is important that a default constructed AqlValue has type VPACK_INLINE and contains the velocypack None type. This is ensure by making all bits in the AqlValue struct zero. Would be good to keep the assertion somewhere.

Reply:
It's kept in erase function, which called from dtor now